packages/core/admin/admin/src/hooks/useFetchClient/index.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { useEffect } from 'react';
+import { getFetchClient } from '../../utils/getFetchClient';
+
+const cancelToken = () => {
+  const CancelToken = axios.CancelToken;
```

Comment:
I think cancel token is deprecated – https://axios-http.com/docs/cancellation

Reply:
Thanks a lot for your suggestion, @joshuaellis